java/test/jmri/jmrix/cmri/CMRISystemConnectionMemoTest.java

```diff
@@ -19,6 +19,25 @@
         Assert.assertNotNull(m);
     }
 
+    @Test
+    public void testNormalizeSystemName() {
```

Comment:
I think this is misnamed. Shouldn't this method be testValidSystemNameFormat instead?